src/main/java/com/weather/controller/WeatherController.java

```diff
@@ -1,18 +1,21 @@
 package com.weather.controller;
 
-import com.weather.model.Weather;
+import com.weather.model.WeatherEntry;
 import com.weather.model.WeatherForecast;
 import com.weather.service.WeatherService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.client.RestTemplate;
 
 
+@Api(description = "Generate Weather Response",produces = "application/json")
 @RestController
 public class WeatherController {
 
-    RestTemplate restTemplate;
+    private RestTemplate restTemplate;
```

Comment:
Seems unused for me with in the class.

- [ ] remove the unused restTemplate.

Reply:
Yeah I forgot to remove since its use is limited to service layer only.